composer.json

```diff
@@ -13,7 +13,10 @@
 	],
 	"require": {
 		"php": ">=5.3.1",
-		"nette/nette": "~2.0"
+		"nette/application": "~2.0",
+		"nette/di": "~2.0",
+		"nette/http": "~2.0",
+		"nette/reflection": "~2.0"
```

Comment:
This dependency is basically useless, the usage in OnResponseHandler::__invoke could be easily replaced by \ReflectionProperty.